_posts/2015-05-28-API-Blueprint-GitHub.md

```diff
@@ -0,0 +1,54 @@
+---
+title: API Blueprint on GitHub
+author: zdenek
+excerpt: "API Blueprint language as a first-class citizen on GitHub"
+layout: post
+date: 2015-05-28 12:00:00
+published: true
+comments: true
+---
+
+Collaboration and continuous integration are the heart of every good API design. This is why we care about integration with CI services and GitHub. I am happy to announce the major milestone for API Blueprint.
+
+API Blueprint is now recognized as a language on GitHub!
+
+All you need to do is to append the `.apib` extension to your blueprint file and push it to GitHub.
+
+## First-class citizen
+
+The growth of API Blueprint over the two years of its existence was tremendous. Adapted by an incredible amount of API developers, consumers, architects and stake holders alike. Today, there are countless number of  blueprints is stored in GitHub. And now, you can search for and see them syntax highlighted!
```

Comment:
`-` Today, there are countless number of  blueprints is stored in GitHub.

`+` Today, there is a countless number of  blueprints stored in GitHub.

Or maybe it just sounds strange to me and it is OK.